src/main/java/frc/robot/Ports.java

```diff
@@ -2,4 +2,11 @@
 
 public final class Ports {
 
+    public static class prototypeArmPorts{
+        public static final int SHOULDER_MOTOR = 22;
+        public static final int ELBOW_MOTOR = 21;
+        public static final int SHOULDER_ENCODER = 22;
+        public static final int ELBOW_ENCODER = 21;
```

Comment:
The encoders are connected through the controller, so you don't need these constants